Dockerfile.NL

```diff
@@ -1,4 +1,4 @@
-FROM pdok/mapserver:7.6.4-patch5-buster-lighttpd
+FROM lighttpd:1.4.65-buster-patch3
```

Comment:
Is this correct @Robbert2407 ?
Shouldn't it be -> pdok/lighttpd:1.4.65-buster-patch3

Reply:
yes it should :-)

Reply:
Most important part was the **/mapserver**
The Docker.NL is based on the main image with a FROM
The base shouldn't be a lighttpd image but a mapserver image

Reply:
you are right i see it my bad